src/utils/__tests__/encoding-test.js

```diff
@@ -78,11 +77,3 @@ describe('base64Utf8Encode', () => {
     expect(result).toBe(expected);
   });
 });
-describe('extractApiKey', () => {
-  test('correctly extracts an API key that has been XORed with a OTP', () => {
-    const key = 'testing';
-    const otp = 'A8348A93A83493';
-    const encoded = xorHexStrings(asciiToHex(key), otp);
-    expect(extractApiKey(encoded, otp)).toBe(key);
-  });
-});
```

Comment:
This test was removed, but doesn't seem to have been replaced in a new location.

Reply:
Thanks. That was intentional, but I should mention it in the commit message. (The test just recites the implementation, and I didn't see value in keeping it. Once `authFromCallbackUrl` gets some tests a couple of commits later, that effectively includes a new test of this code that actually specifies the input.)